GenderPayGap.WebUI/Controllers/SubmitController.CheckData.cs

```diff
@@ -116,6 +116,12 @@ public async Task<IActionResult> CheckData()
                 return new HttpBadRequestResult($"Invalid snapshot year {ReportingOrganisationStartYear.Value}");
             }
 
+            // Don't ask for late reason if the reporting year has been excluded from late flag enforcement (eg. 2019/20 due to COVID-19)
+            if (Global.ReportingStartYearsToExcludeFromLateFlagEnforcement.Contains(stashedReturnViewModel.AccountingDate.Year))
+            {
+                stashedReturnViewModel.ShouldProvideLateReason = false;

```

Comment:
Only need to change this value here, as it determines the action on the green button on the check-data page